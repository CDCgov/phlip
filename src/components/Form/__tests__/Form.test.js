import React from 'react'
import { shallow } from 'enzyme'
import { Form } from '../index'

const props = {
  handleSubmit: jest.fn(),
  form: {},
  role: 'form',
  ariaLabelledBy: 'test-form',
  style: {}
}

describe('Form', () => {
  test('should render correctly', () => {
    expect(shallow(<Form {...props}>form</Form>)).toMatchSnapshot()
  })
})
