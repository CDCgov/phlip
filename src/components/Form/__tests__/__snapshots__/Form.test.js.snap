// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render correctly 1`] = `
<Connect(Form(Form))
  ariaLabelledBy="test-form"
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form={Object {}}
  getFormState={[Function]}
  handleSubmit={[MockFunction]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  role="form"
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  style={Object {}}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
>
  form
</Connect(Form(Form))>
`;
